_examples/complex.env

```diff
@@ -0,0 +1,35 @@
+# Environment Variables
+## ComplexConfig
+## ComplexConfig is an example configuration structure.
+## It contains a few fields with different types of tags.
+## It is trying to cover all the possible cases.
+# Secret is a secret value that is read from a file. (from-file)
+# SECRET=""
+# Password is a password that is read from a file. (from-file, default: '/tmp/password')
+# PASSWORD=/tmp/password
+# Certificate is a certificate that is read from a file. (expand, from-file, default: '${CERTIFICATE_FILE}')
+# CERTIFICATE=${CERTIFICATE_FILE}
+# Key is a secret key. (required)
+# SECRET_KEY=""
+# SecretVal is a secret value. (required, non-empty)
+# SECRET_VAL=""
+# Hosts is a list of hosts. (separated by ':', required)
+# HOSTS=""
+# Words is just a list of words. (comma-separated, from-file, default: 'one,two,three')
+# WORDS=one,two,three
+# Just a comment. (required, default: 'This is a comment.')
+# COMMENT=This is a comment.
+## Anon is an anonymous structure.
```

Comment:
Hmm, been trying to figure out how to get a newline between these two here to separate the last field from the next struct.